### PR TITLE
Fill default values when parent is missing

### LINKED ISSUE #682: When calculating direct features use default value if parent missing
For example, if there is a relationship `transaction.session_id -> sessions.id` and we are calculating a feature `transactions: sessions.SUM(transactions.value)` any rows for which there is no corresponding session should be given the default value of `0` instead of `NaN`.

Of course this should not normally occur, but when it does it seems more reasonable to use the `default_value`.

`DirectFeature.default_value` is already implemented. We should be able to use the same logic that we do for aggregation features.
https://github.com/Featuretools/featuretools/blob/6f4ffd7ef7ea42f95dbaf3892615717a521299db/featuretools/computational_backends/feature_set_calculator.py#L611-L618

### PR DESCRIPTION
**Pull request by thehomebrewnerd**
### Fill default values when parent is missing
Replaces PR #1217 

Closes #682

### CODE REVIEW

File: featuretools/tests/conftest.py [hunk 1]
```diff
@@ -152,6 +152,64 @@ def ks_diamond_es(pd_diamond_es):
     return ft.EntitySet(id=pd_diamond_es.id, entities=entities, relationships=relationships)
 
 
+@pytest.fixture(params=['pd_transactions_defaultvalue_es', 'dask_transactions_defaultvalue_es', 'ks_transactions_defaultvalue_es'])
+def transactions_defaultvalue_es(request):
```
Comment (rwedge) [inline featuretools/tests/conftest.py:156]: thoughts on shortening this to `default_value_es` and adjusting the other fixtures to match?

Comment (thehomebrewnerd) [reply featuretools/tests/conftest.py:156]: Yeah, that would be cleaner for sure.

Comment (thehomebrewnerd) [reply featuretools/tests/conftest.py]: @rwedge Updated

**Review by rwedge — APPROVED** [review]
Looks good!

**Pull request merged**